components/navbar/style.css

```diff
@@ -0,0 +1,81 @@
+header {
+    background-color: #036;
+    border-bottom: 2px solid #fcba19;
+    padding: 0 30px 0 30px;
+    color: #fff;
+    display: flex;
+    height: 65px;
+    top: 0px;
+    position: fixed;
+    width: 100%;
+    -webkit-box-shadow: 0 6px 8px -4px #b3b1b3;
+    -moz-box-shadow: 0 6px 8px -4px #b3b1b3;
+    box-shadow: 0 6px 8px -4px #b3b1b3;
+  }
+  
+  header h1 {
+    font-family: 'PT Sans';
+    font-weight: normal;  /* 400 */
+    margin: 5px 5px 0 18px;
+    visibility: hidden;
```

Comment:
is the h1 title dropped?

Reply:
Its hidden for small screens but visible otherwise.